### DISCUSSION TITLE
Access Figma File

### DISCUSSION THREAD

**Discussion by Happiartista** [3Stones-io/omedis#399] [General]
Here's is a link to access the current Figma file. Its currently set to View but you can also leave notes or comments - https://www.figma.com/design/y0zOV0u8V4nuBapCzgUcKn/OMEDIS?node-id=36-84&t=OF510eSgdNNldAIe-1
@almirsarajcic @wintermeyer @amos-kibet @kagure-nyakio .

**Comment by almirsarajcic** [comment]
Thanks, @Happiartista!
